scene/animation/tween.h

```diff
@@ -161,6 +162,8 @@ class Tween : public RefCounted {
 	TweenProcessMode get_process_mode();
 	Ref<Tween> set_pause_mode(TweenPauseMode p_mode);
 	TweenPauseMode get_pause_mode();
+	Ref<Tween> set_ignore_time_scale(bool p_ignore);
+	bool is_ignore_time_scale() const;
```

Comment:
In a similar PR that does the same thing to `Timer` nodes, we went with `get_ignore_time_scale` instead of `is_ignore_time_scale`. See discussion here : https://github.com/godotengine/godot/pull/96626#discussion_r1889253817
It would make sense to do the same here

Reply:
This is basically bikeshedding but I'd not be against `is_ignoring_time_scale()` myself for both.

Reply:
I agree with the points in the discussion, but this method is not exposed, much like the one in `SceneTreeTimer`.

From the other tween ignore time pull https://github.com/godotengine/godot/pull/100736 there is talk about general code cleanup in Tween. I think it would be a good idea to change `is_ignore_time_scale` both here and in `SceneTreeTimer` for consistency, alongside the other code polish.

I can implement it, but I'm not sure if this PR should encompass all that.

Reply:
I think it should be `is_ignoring_time_scale` that's our usual formatting 

Also this should be bound as well, doesn't make sense to bind only one

Reply:
If we feel particularly strongly about it you could make a separate PR for Timer's before we no longer can @AThousandShips

Reply:
Absolutely will see if I can get that today